lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :dishes, :report_price
```

Comment:
:dishes exposes dishes to modification because it is a hash. :report_price is not an instance variable.